docs/linear_compiler.md

```
@@ -0,0 +1,235 @@
+config = LinearCompilerConfig(hardware_timing=timing)
+compiler = LinearCompiler(architecture, config=config)
+```
+
+`rx`, `ry`, and `rz` may be selected as virtual single-qubit gates through
+`GateTiming.virtual_single_qubit_gates`. Virtual rotations must have zero
+duration and remain in the logical schedule without reserving hardware time.
+`rz` is virtual by default, reflecting a common trapped-ion implementation.
+
+## Choose a search profile
+
+The compiler uses A* search: it compares the time already used with an estimate
```

markdownlint-cli2 (0.23.2)

[warning] Spaces inside emphasis markers

(MD037, no-space-in-emphasis)